test/assignment.coffee

```diff
@@ -141,9 +141,6 @@ test "#1192: assignment starting with object literals", ->
 
 # Destructuring Assignment
 
-test "empty destructuring assignment", ->
-  {} = [] = undefined
-
```

Comment:
@connec removed that, but I suspect because it’s [invalid ES](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=true&presets=&targets=&browsers=&builtIns=false&code=%7B%7D%20%3D%20%5B%5D%20%3D%20undefined).